registry/coder/modules/windows-rdp/README.md

```diff
@@ -46,7 +46,7 @@ module "windows_rdp" {
 module "windows_rdp" {
   count       = data.coder_workspace.me.start_count
   source      = "registry.coder.com/coder/windows-rdp/coder"
-  version     = "1.0.18"
+  version = "1.2.0"
```

Comment:
@DevelopmentCats there is a bug and the script is burning the version by two minor versions.